tools/deb/istio-iptables.sh

```diff
@@ -176,6 +191,9 @@ if [ -n "${IP_RANGES_INCLUDE:-}" ]; then
     done
     iptables -t nat -A ISTIO_OUTPUT -j RETURN
 else
-    iptables -t nat -A ISTIO_OUTPUT -j ISTIO_REDIRECT
+    if [ "${K8S_MODE}" == "1" ]; then
+       iptables -t nat -A ISTIO_OUTPUT -j ISTIO_REDIRECT
+    fi
+    # Else: don't intercept anything, to avoid breaking the VM in case Istio is not configured properly
```

Comment:
echo ...

Reply:
See above - the purpose of this change is to be more conservative on the raw vm, while
keeping the behavior in k8s.